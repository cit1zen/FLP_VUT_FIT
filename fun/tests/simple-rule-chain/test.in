A,B,C,D
a,b,c
A
A->aaB
A->ccB
B->bB
B->#
C->B
A->D
D->C
