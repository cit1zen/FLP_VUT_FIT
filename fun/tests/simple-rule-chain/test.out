A,A2,A3,B,C,D
a,b,c
A
A->#
A->aA3
A->bB
A->cA2
A2->cB
A3->aB
B->#
B->bB
C->#
C->bB
D->#
D->bB
